api/alias_api/__init__.py

```diff
@@ -56,19 +60,16 @@ def import_alias_api():
     if not os.path.exists(module_path):
         return
 
-    # finally, import the alias_api module directly from the pyd file
-    if sys.version_info.major == 3:
-        import importlib.util
-        spec = importlib.util.spec_from_file_location(module_name, module_path)
-        alias_api = importlib.util.module_from_spec(spec)
-        spec.loader.exec_module(alias_api)
-    else:
-        import imp
-        alias_api = imp.load_source("alias_api", module_path)
+    spec = importlib.util.spec_from_file_location(module_name, module_path)
+    alias_api = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(alias_api)
 
     # add the newly created module oject to sys.modules and remap the globals accessor to point at our new module
     sys.modules["alias_api"] = alias_api
     globals()["alias_api"] = sys.modules["alias_api"]
 
 
+if sys.version_info.major < 3:
+    raise Exception("Alias Python API supports Python version >= 3. Using Python version {}".format(sys.version))
```

Comment:
"The Alias Python API now only supports Python 3. You are using version {} of Python. Please refer to this [page](https://github.com/shotgunsoftware/tk-alias/wiki/Python-Version-Support) for additional information."

Reply:
Updated error message now looks like this:
<img width="477" alt="api-python2-error-edited" src="https://user-images.githubusercontent.com/66642500/162815257-2ce366ba-aa3c-40c4-abe3-7c0f896f8935.png">